### PR TITLE
Fixed URLs for HTML, CSS, JS

### PR DESCRIPTION
**Pull request by sushant4191**
Fixed the redirection of URL for HTML, CSS and Javascript

### CODE REVIEW

**Comment by sushant4191** [comment]
@saragibby Please take a look at these translation changes too.

File: 3-terrarium/translations/README.hi.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 # पाठ
 
-1. [एचटीएमएल का परिचय](../1-intro-to-html/README.hi.md)
-2. [सीएसएस का परिचय](../2-intro-to-css/README.hi.md)
-3. [डोम और जेएस क्लोजर का परिचय](../3-intro-to-DOM-and-closures/README.hi.md)
+1. [एचटीएमएल का परिचय](../1-intro-to-html/translations/README.hi.md)1-intro-to-htmlREADME.hi.md
```
Comment (saragibby) [inline 3-terrarium/translations/README.hi.md:9]: It looks like there is some text to be removed at the end of this line.

**Review by saragibby — CHANGES_REQUESTED** [review]
Thank you for making these updates! Found a small item to fix and then we can get this approved and merged,

File: 3-terrarium/translations/README.hi.md (hunk 1 above)
Comment (sushant4191) [reply 3-terrarium/translations/README.hi.md]: Thanks for noticing!

**Review by saragibby — APPROVED** [review]
Looks great, thanks for getting this updated!

**Pull request merged**